<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnConnectSever">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>连接服务器</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>771</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string>聊天室</string>
    </property>
    <widget class="QPlainTextEdit" name="editMessage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>581</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sendMessage">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>390</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="editSendMessage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>581</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelSeverStatus">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>30</y>
       <width>411</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>status: 无连接</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="flag">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>50</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
